garden/ros_gz_project_template_guide.md

```diff
@@ -0,0 +1,97 @@
+# Guide to `ros_gz_project_template` for ROS 2 and Gazebo Development
+
+In this guide, you will learn how to use the `ros_gz_project_template` to create a (recommended) structured workspace or improve your existing workspace for your ROS 2 and Gazebo projects. This template offers a consistent layout, automated build process, and integration with both ROS 2 and Gazebo, enabling you to focus on developing your robotics applications.
+
+## Installation Steps
+
+1. If you are starting a project, create new workspace or go to your existing project's source directory:
+
+```bash
+    mkdir -p ~/project_ws/src
+    cd ~/project_ws/src
+```
+
+1. Directly `Use this template` and create your project repository on Github.
+
+   ![use_template](tutorials/ros2_integration/use_template.png)
+
+   Or start by cloning the `ros_gz_project_template` repository:
+
+   ```bash
+    wget https://raw.githubusercontent.com/gazebosim/ros_gz_project_template/main/template_workspace.yaml
+    vcs import < template_workspace.yaml
+   ```
+
+1. Rename the cloned repository folder to your desired project name:
+
+   ```bash
+   mv ros_gz_project_template your_project_name
+   ```
+
+   Change your working directory to the project workspace:
+
+   ```bash
+   cd ~/project_ws
+   ```
+
+## Package structure
+
+At this point you'll have the following packages in your project workspace:
```

Comment:
Maybe not say workspace, since workspace refers to the `project_ws` directory, and these bullets are actually in the repo itself in `project_ws/src/your_project_name`. The workspace also has more than the ones in the list - it has sdformat_urdf etc, so saying workspace here could be confusing
```suggestion
At this point you'll have the following packages in your project:
```